src/sync-client.js

```diff
@@ -910,6 +911,11 @@ var self = {
 
   doCloudCall: function(params, success, failure) {
     if(self.cloudHandler && typeof self.cloudHandler === "function" ){
+      if (params && params.req && params.req.__fh) {
```

Comment:
I don't think `params.req.__fh` will ever be true as we don't have any other places define `params.req.__fh`?